meetings/working-groups/discriminated-unions/conceptual-overview.md

```diff
@@ -0,0 +1,55 @@
+# Conceptual overview
+
+> The text that follows is the working draft for how we'll introduce this concept to customers. It's meant to address two distinct challenges:
+>
+> - Many of the scenarios for unions can also be handled by inheritance. That prompts the question on the value of adding unions to C#.
+>
+> - Unions have different meanings in different computer languages. There are tagged unions, type unions, C-style (unsafe) unions. Which do we mean?
+>
+> The language is intended to be less formal than the specification and serve as an introduction for customers.
+
+*Discriminated unions* provide a new way to express the concept that *an expression may be one of a fixed set of types or values*.
+
+C# already provides syntax for special cases of discriminated unions. Nullable types express one example: A `int?` is either an `int`, or `null`. A `string?` is either a `string` value or `null`. Another example is `enum` types. An `enum` type represents a closed set of values. The expression is limited to one of those values.
```

Comment:
Again, is "closed" the right word, as it seems to indicate that only values that are enums can be held in enums. 

Examples are good, this is just a comment on wording